src/Microsoft.DotNet.Tools.Pack/ProjectBuilder.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Text;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+
+namespace Microsoft.DotNet.Tools.Pack
+{
+    internal class ProjectBuilder
```

Comment:
The Builder verb sounds like StringBuilder... a thing that itself does building. This is a thing that invokes the Build command. BuildProjectCommand.Execute()?